plugins/github-actions/src/api/types.ts

```diff
@@ -49,3 +49,25 @@ export enum BuildStatus {
   'pending',
   'running',
 }
+
+/** @public */
+export type Branch = {
+  name: string;
+};
+
+/** @public */
+export type Branches = {
+  default_branch: string;
+  branches: Branch[];
+};
+
+/** @public */
+export enum ViewEnum {
+  Table = 'table',
+  Card = 'cards',
+}
+
+/** @public */
+export interface RouterProps {
+  view?: ViewEnum;
```

Comment:
```suggestion
  view?: 'cards' | 'table';
```

Let's avoid additional exports for the public API. It's fine to use an enum internally if you prefer, but let's avoid having the integrator import and extra symbol to configure this.